src/app/data/routes.js

```diff
@@ -166,6 +169,13 @@ export const routes = [
 		Component: Admin,
 		condition: true,
 	},
+	{
+		name: '/products',
+		title: __( 'My Products', 'wp-plugin-bluehost' ),
+		Component: Products,
+		Icon: BluehostIcon,
```

Comment:
The page icon (Bluehost) is white. The icon should match the rest of the icons. Try: 
`import { Squares2X2Icon } from '@heroicons/react/24/outline';`